bubblegum/program/download-compression-programs.sh

```diff
@@ -4,19 +4,17 @@
 set -e
 
 MPL_ROOT=$(git rev-parse --show-toplevel)
-mkdir -p $MPL_ROOT/target/deploy
+mkdir -p $MPL_ROOT/test-programs
 
 mkdir -p solana_program_library
-curl -LkSs https://api.github.com/repos/solana-labs/solana-program-library/tarball/5f189d1fc0722a619c9b850f2769f85fccb5baea | tar -xz --strip-components=1 -C ./solana_program_library
+curl -LkSs https://api.github.com/repos/solana-labs/solana-program-library/tarball/b0d84fc4 | tar -xz --strip-components=1 -C ./solana_program_library
```

Comment:
This probably should be updated to 6e8179484413cd8a50ee794c0a49cfef91e1fcb66e8179484413cd8a50ee794c0a49cfef91e1fcb6 or newer